### ISSUE TITLE
It appears as normal drawer menu 

### ISSUE THREAD

**Issue by mohamed-ali-dbs** [AppCheap/awesome_drawer_bar#4]
wrote the code exactly as the example, except i use getx to manage the state

**Comment by jtkeyva** [comment]
yes, it looks like a regular drawer. do you have a better example that matches your demo images?

**Comment by damamire** [comment]
anyone got this to work?

**Comment by chitraarasu** [comment]
This package not working as expected use [flutter_zoom_drawer](https://pub.dev/packages/flutter_zoom_drawer)
